orm/entities/SubmissionVersion/TallySheetVersion/TallySheetVersionPRE21.py

```diff
@@ -86,9 +89,16 @@ def html(self):
             if row.count is not None:
                 content["totalRejected"] = content["totalRejected"] + row.count
 
+        # convert image to data-uri
+        data_image = ''
+
+        with open("static/Emblem_of_Sri_Lanka.png", "rb") as img_file:
+            data_image = base64.b64encode(img_file.read()).decode()
+
         html = render_template(
             'PRE-21.html',
-            content=content
+            content=content,
+            data_image=data_image
```

Comment:
Add the logo within content rather having this as a separate parameter

Reply:
updated in https://github.com/ECLK/results-tabulation-api/pull/348